### ISSUE TITLE
nodejs usage causes navigator error

### ISSUE THREAD

**Issue by CallumJFraser** [kaimallea/isMobile#69]
when trying to run ismobile as an npm module in nodejs there is an error returned:

```
ReferenceError: navigator is not defined
    at IsMobileClass (/Users/aUser/Projects/sanitycheck/node_modules/ismobilejs/isMobile.js:58:31)
    at Object.<anonymous> (/Users/aUser/Projects/sanitycheck/index.js:3:1)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

**Comment by YaoaY** [comment]
maybe you need pass `user-agent` by yourself:
```
var isMobile = require('ismobilejs');
console.log(isMobile(req.headers['user-agent']));
```

**Comment by kaimallea** [comment]
Fixed by #77
